operations/filter_kgraph_top_n.yml

```diff
@@ -0,0 +1,49 @@
+id: filter_kgraph_top_n
+subclass_of: filter_kgraph
+name: filter kgraph (top n)
+description: This operation removes kgraph edges that have attribute values are below/above the top/bottom n values.
+examples:
+- input: fill_and_filter/messages/05_filtered_kgraph_stat_input.json
+  output: fill_and_filter/messages/08_filtered_kgraph_top_n_output.json
+  parameters:
+    edge_attribute: normalized_google_distance 
+    threshold: 3
+    top: false
+    qnode_keys: 
+      - drug
+input_requirements: []
+output_guarantees:
+- no edges in the kgraph which have attribute values are below/above the top/bottom n values.
+allowed_changes:
+- remove knodes
+- remove kedges
+parameters:
+  type: object
+  properties:
+    edge_attribute:
+      description: The name of the edge attribute to filter on.
+      type: string
+      example: normalized_google_distance
+    max_edges:
+      description: The number of edges to keep.
+      type: integer
+      example: 10
+      minimum: 0
+      default: 50
+    top:
```

Comment:
The description is clear, but it would be nice if the name alone were more informative. I have a hard time naming boolean flags. What if we did something like this?
```
keep_top_or_bottom:
  type: string
  enum:
  - top
  - bottom
  default: top
```
It's a little verbose maybe, but clearer hopefully.

Reply:
I would be fine with that. @dkoslicki do you have an opinion on making this name a bit more verbose? We would also want to change this in main for std_dev since that was already merged.